### PR TITLE
feat: Trash/Delete links

### PR DESCRIPTION
**Pull request by jameshoulahan**

### CODE REVIEW

**Review by xmichelo — APPROVED** [review]

File: link_folder.go [hunk 1]
```diff
@@ -20,16 +22,72 @@ func (c *Client) ListChildren(ctx context.Context, shareID, linkID string) ([]Li
...
+
+		if err := c.do(ctx, func(r *resty.Request) (*resty.Response, error) {
+			return r.SetBody(req).SetResult(&res).Get("/drive/shares/" + shareID + "/folders/" + linkID + "/trash_multiple")
+		}); err != nil {
+			return err
+		}
+
+		for _, res := range res.Responses.Responses {
+			if res.LinkResponse.Response.Code != SuccessCode {
+				return fmt.Errorf("failed to import message: %w", res.LinkResponse.Response)
```
Comment (scop) [inline link_folder.go:52]: This seems a bit off for someone glancing the code for the first time, perhaps `failed to trash children: %w` instead?

File: link_folder.go [hunk 2]
```diff
@@ -20,16 +22,72 @@ func (c *Client) ListChildren(ctx context.Context, shareID, linkID string) ([]Li
...
+
+		if err := c.do(ctx, func(r *resty.Request) (*resty.Response, error) {
+			return r.SetBody(req).SetResult(&res).Get("/drive/shares/" + shareID + "/folders/" + linkID + "/delete_multiple")
+		}); err != nil {
+			return err
+		}
+
+		for _, res := range res.Responses.Responses {
+			if res.LinkResponse.Response.Code != SuccessCode {
+				return fmt.Errorf("failed to import message: %w", res.LinkResponse.Response)
```
Comment (scop) [inline link_folder.go:87]: `failed to delete children: %w`?

File: link_folder.go (hunk 1 above)
Comment (jameshoulahan) [reply link_folder.go:52]: Good catch, copy paste mistake from the message import method, thanks!

File: link_folder.go (hunk 2 above)
Comment (jameshoulahan) [reply link_folder.go:87]: Thanks, copy paste mistake :) fixed in latest PR

**Pull request merged**